articles/active-directory/develop/troubleshoot-publisher-verification.md

```diff
@@ -186,6 +186,12 @@ The target application (`AppId`) can’t be found. Provide a valid application I
     
 Most commonly caused when verification is being performed via Graph API, and the ID of the application provided is incorrect. Note- the ID of the application must be provided, not the AppId/ClientId.
 
+### ApplicationObjectisInvalid 
+
+The target application (AppId) is not valid. Please provide a valid application ID and try again. 
+
+Most commonly caused when the verification is being performed via Graph API, and the ID of the application provided does not exist. Note-the ID of the application must be provided, not the AppId/ClientId. 
```

Comment:
What's the difference between AppId and the ID of the application? It might be helpful to attach a doc that identifies what the ID of the application is vs. the ID of the application
@obrianokoyo